examples/erc721-consecutive/src/constructor.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
```

Comment:
As mentioned in a previous comment, we either don't need this file or we don't need an `init()` function.